toile-cli/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "toile-cli"
+version = "0.1.0"
+authors = ["Sam Wilson <sam.wilson@consensys.net>"]
```

Comment:
If this is meant to be the repo where we store all the combined work, probably makes sense to add me and Ansgar to all these toml files also.

Reply:
What email addresses would you and @adietrichs like me to include?

Reply:
Just my consensys one works, assuming same for Ansgar?  Mine is gregory.trowbridge@consensys